### ISSUE TITLE
Weekly Atlas Meeting Log

### ISSUE THREAD

**Issue by bedeho** [Joystream/atlas#818]
# Meeting Date

Monday, June 21st, 2021.

# Participants

- Bedeho
- Klaudiusz
- Milosz
- Adam
- Kuba

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-864880007
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-864874625

# Agenda

1. Review where we are.
2. Land rough scope for firs sprints.
3. Orion data issue
4. Some coordination issues
5. What do do when Klauidusz si gone
6. Fix Figma.

# Todo

- Bedeho will review the player issue to double check the scope.
- Bedeho needs to review video presentation pages, and possibly have follow up with Klaudusz
- Bedeho needs to review new issue from Klausiusz: https://github.com/Joystream/atlas/issues/132, stiching node.
- Adam will breakdown the player with estimates.
- Kuba will share a prototype tomorrow with discovery format a video format!
- Kuba and Milosz will review backlog of UI bugs/issues in Github to close out already fixed problems
- Kuba and Milosz will review all of your pending work, and suggest a first sprint with a specific scope and deadine asap.
- Tech team will finish the tech debt in 3-4 days.
- Designers will introduce status indication on all pages Figma, and introduce new issues on Github  tracking overall progress on certain screen, and share with Bedeho.
- Based on status indication review from Kuba & Milosz, and breakdown of issues from ADam, Klaudiusz & Bedeho will suggest a larger scale sequence of Sprints for the tech team.

**Comment by bedeho** [comment]
# Meeting Date

Monday, June 28th, 2021.

# Participants

- Bedeho
- Klaudiusz
- Milosz
- Adam
- Kuba

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-868687999
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-868797014

# Agenda

- Proposition: All sprints must be organized as Github milestones, if you are not working off a milestone, you are doing it wrong. We still have no milestones-
- Review status of TODOs from last week: https://github.com/Joystream/atlas/issues/818#issue-926098885
- (hyper speed, <2m) Review conclusion about discovery design: https://github.com/Joystream/atlas/issues/878
- (hyper speed, <2m) Review feedback to search & discover: https://github.com/Joystream/atlas/issues/874
- Review actual issues or problems
  - Scrubbing issue: https://github.com/Joystream/atlas/issues/785#issuecomment-868990362
  - Bad view of slow asset download: https://github.com/Joystream/atlas/issues/877
  - Avatar clear: https://github.com/Joystream/atlas/issues/737
  - Upload screen fix: https://github.com/Joystream/atlas/issues/790
  - In-app Feedback: https://github.com/Joystream/atlas/issues/879
- Review implications for design team from decision on what tech team should do next.
- Bedeho+Klaudius Questions:
  - What is the status of the player component?
  - How to signal status in Figma?
  - What is the status of scaling of videos and channel previews?
  - What is the status of the player page (not only player)?
- What is the status of getting people into sprints?
  - Adam: Player
  - Tech: ?
  - Kuba&Milosz: ?


# Todo

- There will be the following sprints
  - **Player Component Design:** which is only including Adam, expected delivery time for that is end of this week
  - **Player MVP Implementation:** which is the tech side of implementing the MVP part of the player designed, as scoped by Klaudiusz. Klaudiusz will provide early feedback to Adam for prioritization features. Expected delivery < 3 weeks. Tech team will be one dedicated person.
  - **Video Content Page:** design and implementation of the video content pages. Unclear delivery time. **High priority** for Kuba to unlock for tech team, which will be one dedicated person. Some tech work required on Orion for new API calls. Possibly also some work on the curator CLI and query node schemas? (Klaudiusz will identify)
  - **Channel update:** design and implementation of new channel views, on consumer and studio side. Very little work required on design side, only some responsiveness and small details (search, follow management modal).
- Klaudiusz will make Github Milestone for all sprints.
- Klaudiusz will make one global Github Project for all issues.
- Klaudiusz will make an issue that explains what exactly the browser allows us to know about the state of the loading of an image, as well as controlling what is displayed. This is as a precursor to redesigning the rendering policy for previews, as in this issue https://github.com/Joystream/atlas/issues/877. This design work will not be started now.
- Everyone (including bedeho) will meet later in the week to discuss a final iteration, proposed by Kuba, for how to deal with presentation of video previews, channel previews, playlists and other items in a responsive grid.
- Kuba and Milosz will review backlog of UI bugs/issues in Github to close out already fixed problems.
- We will do hotjar for in-app feedback at a later time.

# Non-action Conclusions

- We postponed introducing status indicators on Figma pages.
- We postponed any further work on upload area.
- We postponed any further work on upload status toasts.

**Comment by bedeho** [comment]
# Meeting Date

Monday, July 5th, 2021.

# Participants

- Bedeho
- Diego
- Milosz
- Adam
- Kuba

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-873089823
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-873425718

# Agenda

- Remark about adding the following to estimates: `For any sprint where author is the sprint master: Total hours across all estimated issues that are not done. (no partial progress counting), and equivalent estimated completion day based on team size working on it.`
- Review status of TODOs from last week: https://github.com/Joystream/atlas/issues/818#issuecomment-868994206
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
  - what is tech team working on, what is design team working on?
  - are milestones live?
  - are issues created?
  - is progress statuss accurate?
  - completion estimates?

# Todo

- Kuba will share approach to responsive preview of videos, channels, playlists, etc. by end of week.
- Bedeho will try to provide input to Adam about priorities for next week, but if that is not feasible, Adam will simply pick other high value backlog tasks.

# Non-action Conclusions

- Kuba is away next week.

**Comment by bedeho** [comment]
# Meeting Date

Monday, July 19th, 2021.

# Participants

- Bedeho
- Klaudiusz
- Milosz
- Adam
- Kuba

# Weekly Summaries

No summaries were submitted for this past week, but the week before this we had the following ones

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-877379404
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-877705046

# Agenda

- Review status of TODOs from last week: https://github.com/Joystream/atlas/issues/818#issuecomment-873656309
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
  - is everyone on one or more milestones?
  - what is the current ETA on those milestones?
- We urgently need a small sprint for this issue: https://github.com/Joystream/atlas/issues/782
- When is a suitable time for design team to start tackling large major new product features: NFTs, comments, social tokens, etc.
- Atlas specific issues (only Bedeho & Klaudiusz needed)
   - Graceful playback failure:  https://github.com/Joystream/atlas/issues/1002
   - Atlas error recording: https://github.com/Joystream/atlas/issues/1003
   - Distinct error state for encoding vs infrastructure: https://github.com/Joystream/atlas/issues/1004

# Todo

- Bedeho will complete review of oustanding work Kuba has done on `Video tiles + Title area + Channel`.
- Klaudiusz will take on a set of related issues having to do with dealing with failure in the infrastructure better:
  - https://github.com/Joystream/atlas/issues/1003
  - https://github.com/Joystream/atlas/issues/1002
  - https://github.com/Joystream/atlas/issues/1004: here the research is the highest priority, but if its practical, he will ask Adam to do some additional design to put it in production
- Klaudiusz will prepare a weekly summary based on current status, in place of missing summary from last week

# Non-action Conclusions

- Adam is off next week.
- Kuba may be finished with all current sprints as early as next week, at which points the following categories of tasks can be tackled
  - Better `My Uploads` & dealing with upload quota limits in uploading flow.
  - Search experience
  - UpdateVideo player Page
  - Discovery Page
  - Following Page
  - Watchlist
  - Totally new things: NFTs, comments, social tokens, etc.
- **Milestones estimates**
  - Channel view update (@eldiegod) - end of week (23.07)
  - Video player update (@drillprop) - end of week (23.07)
  - Video content pages (@wyrdrender) - end of next week (30.07)
  - Misc hotfixes (@kdembler) (asset fallbacks, asset monitoring, better errors for video player) - end of week (23.07)

**Comment by bedeho** [comment]
# Meeting Date

Monday, July 26th, 2021.

# Participants

- Bedeho
- Klaudiusz
- Milosz
- ~Adam~
- Kuba

# Weekly Summaries

No summaries were submitted for this past week, but the week before this we had the following ones

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-886485406
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-886472088

# Agenda

- Review status of TODOs from last week: https://github.com/Joystream/atlas/issues/818#issuecomment-882118268
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- When will new work go into production?
- A large share of the tech team is will be idle this week. This is somewhat problematic, as its not clear we have many high value tasks laying around, and the lead time to get new work from the designers is quite long. **What can we do?**
  - What is likely lead time until design team is done with new major
  - Klaudiusz has suggested some minor tasks
- Next design sprint for Kuba: Based on current feedback from users, and also lack of preparation of new work, lets try tackle a better `My Uploads` screen, as well as in general improving how well the product deals with various upload failures.

# Todo

- Klaudiusz will prepare a single snapshot by the end of the week for production release.
- Klaudiusz will plan a new sprint for introducing server side error capture with Elastics stack.
- Kuba will do a review of the final snapshot to be released.
- Kuba will finish all sprints tomorrow.
- Bedeho will prepare some notes on what can be done to improve `My Uploads`
- Kuba & Bedeho have meeting on Wednesday to
  a) process backlog of design issues and categorise them.
  b) review search as a possible next area of work

# Non-action Conclusions

- Klauiudsz expects 1-2 weeks you think we will be fine in terms of work for tech team.

**Comment by bedeho** [comment]
# Meeting Date

Monday, July 28th, 2021.

# Participants

- Bedeho
- Klaudiusz
- Kuba

# Agenda

Review all outstanding `design` issues, and attempt to close or tackle as many as possible, as well as nail down possible next sprints.

# Todo

Here are the major sprint candidates identified and order in decreasing ROI:

1. **Uploading Flow:** https://github.com/Joystream/atlas/issues/1063
2. **Discovery:** https://github.com/Joystream/atlas/issues/874
3. **Search:** https://github.com/Joystream/atlas/issues/874
4. **Grid Layout:** https://github.com/Joystream/atlas/issues/889
5. **User Input:**
   - https://github.com/Joystream/atlas/issues/715
   - https://github.com/Joystream/atlas/issues/861
 6. **Misc:** Can probably be somewhat more constrained in scope
    - https://github.com/Joystream/atlas/issues/717
    - https://github.com/Joystream/atlas/issues/642 + https://github.com/Joystream/atlas/issues/794
    - https://github.com/Joystream/atlas/issues/856
    - https://github.com/Joystream/atlas/issues/858
    - https://github.com/Joystream/atlas/issues/859
    - https://github.com/Joystream/atlas/issues/869
    - https://github.com/Joystream/atlas/issues/872
    - https://github.com/Joystream/atlas/issues/890
    - https://github.com/Joystream/atlas/issues/892

**Comment by bedeho** [comment]
# Meeting Date

Monday, Aug 2nd, 2021.

# Participants

- Bedeho
- Klaudiusz
- Kuba

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-890308236
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-890306073

# Agenda

- Review status of TODOs from last week: https://github.com/Joystream/atlas/issues/818#issuecomment-888439529
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- Resolve outstanding problem with video pages sprint, and select solution.
- Briefly discuss RWD approach taken her: https://github.com/Joystream/atlas/issues/893#issuecomment-887682778
- Determine what sprints to tackle next for different parts of the team, using sprint menu identified here: https://github.com/Joystream/atlas/issues/818#issuecomment-888439529

# Todo

- Kuba will try to find a simpler solution for showing channels that are relevant to the user that does not require introducing any changes on the query node layer.
- Adam should review player in production when back tomorrow, and compare with designs.
- Klaudiusz will write down his analysis of the current upload problems in a standalone issue.
- Kuba+Klaudiusz will start a new sprint which will improve the UX of the upload flow experience, each contributing the design and tech issues required, with estimates. We will tackle upload limit notification, snackbar notifications and mid-flow dialogs in the same sprint.
- Adam starts on Discovery sprint when back, and will create require design issues.
- Diego starts on a pure tech Grid layout sprint.
- Bartosz will join Rafal on video content pages sprint, and when they are finished, they will both transition to work on the Discovery sprint.

**Comment by bedeho** [comment]
# Meeting Date

Monday, Aug 2nd, 2021.

# Participants

- Bedeho
- Klaudiusz
- Kuba

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-894339750
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-894361331

# Agenda

- Review status of TODOs from last week.
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- Review new Gantt chart for inter-temporal coordination, and update based on new information.

# Todo

- Bedeho will fix event to be recurring.
- Adam will create required design issues for Discovery sprint.
- Adam will come up with list of requirements for analytics, and make Github issue.
- Klaudusz will consider creating document in Atlas repo for onboarding new contributors by describing all relevant processes we are using.

# Non-action Conclusions

- Video content pages may be delayed another halfway next week.
- Milosz will prepare proposal for agenda and plan for a team wide review meeting to ensure future collaboration is as effective as possible.
- New process: When designers make small ad-hoc work, not scheduled, they should
  - create a new issue
  - move the issue to the sprint board on Github in `Todo`
  - add `tech` label, and notify klaudiusz

**Comment by bedeho** [comment]
# Meeting Date

Monday, Aug 16th, 2021.

# Participants

- Bedeho
- Klaudiusz
- Kuba
- Adam

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-898589146
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-898536100

# Agenda

- Review status of TODOs from last week.
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- Review proposal about future of curation: https://github.com/Joystream/joystream/issues/2570
- Review how to provide data for featured content in categories in new design.
- Review new Gantt chart for inter-temporal coordination, and update based on new information.

# Todo

- Kuba will review list of analytics requirements suggested by Adam, and they will share with rest of team.
- Klaudiusz & Adam will have a Figma reorganisation meeting later this week.
- Bedeho & Klaudiusz will have a meeting on Thursday to to discuss prospects of introducing the early stage of the gateway node.

# Non-action Conclusions

The way we expect to organise work in the next two weeks looks as follows.

![Screenshot 2021-08-16 at 12 37 05](https://user-images.githubusercontent.com/437292/129551038-b194600e-18ce-4720-8273-23ca9f021c5d.png)

**Comment by bedeho** [comment]
# Meeting Date

Monday, Aug 23rd, 2021.

# Participants

- Bedeho
- Klaudiusz
- Kuba
- Adam

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-902802397
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-902820720

# Agenda

- Review status of TODOs from last week.
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- Review new Gantt chart for inter-temporal coordination, and update based on new information.
- Do we need to introduce designer Q&A review for all product releases?
- Can we start doing this: https://github.com/Joystream/atlas/issues/1227

# Todo

- Adam will share new iteration of discover page with Bedeho, requirement for urgent review.
- Kuba will share new iteration of upload flow design, requirement for urgent review.

# Non-action Conclusions

- We will start doing more continuous review by the designers of the work that the tech team is doing, and it will be the responsibility of the designers to initiate and stay on top of these. Tech team should feel free to get in touch with designers at any time during sprints.
- Klaudiusz will try to announce releases in advance to the design team & Bedeho to make everyone aware of changes to the production deployment, and possible last minute review.
- We are not doing anything to introduce filtering out of empty channels at this time, it will be added to the `Giza` release
- Here is the updated Gantt chart

![Screenshot 2021-08-23 at 12 55 57](https://user-images.githubusercontent.com/437292/130436137-466361ec-37f1-468f-b580-5a6b6369df90.png)


# Expected Releases

- Minor tweaks and improvements to the video content pages.
- Additional metrics for logging and assets

**Comment by bedeho** [comment]
# Meeting Date

Monday, Aug 30th, 2021.

# Participants

- Bedeho
- Klaudiusz
- Kuba
- Adam

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-907749962
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-907342192

# Agenda

- Review status of TODOs from last week.
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- Review new Gantt chart for inter-temporal coordination, and update based on new information.
- Do we need to introduce designer Q&A review for all product releases?
- Proposal: In order to facilitate a shared understanding of design sprints, or the design aspect of a sprint, we should have the design lead prepare a scope brief, in the form of a Github issue, which describes what we are trying to tackle within a given sprint.
- The current improved upload flow design has stage where it tries to warn the user that they are out of storage space. This however requires that query node exposes information about this. This will take a considerable amount of time to do. An alterntaive could therefore be to bundle this work in with the upcoming Giza release, and either not implement this part of the improvements in this sprint, or to implement a simpler version which does not try to give the detailed information presented in the dialog, merely some sort of dialog that explains that the publishing failed due to a specific reason.

# Todo

`Fill in after meeting`

# Non-action Conclusions

`Fill in after meeting`

# Expected Releases

`Fill in after meeting`

**Comment by bedeho** [comment]
# Meeting Date

Monday, September 1st, 2021.

# Participants

- Bedeho
- Klaudiusz
- Kuba
- Adam

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-907749962
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-907342192

# Agenda

- Review status of TODOs from last week.
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- Review new Gantt chart for inter-temporal coordination, and update based on new information.
- Do we need to introduce designer Q&A review for all product releases?
- Proposal: In order to facilitate a shared understanding of design sprints, or the design aspect of a sprint, we should have the design lead prepare a scope brief, in the form of a Github issue, which describes what we are trying to tackle within a given sprint.
- The current improved upload flow design has stage where it tries to warn the user that they are out of storage space. This however requires that query node exposes information about this. This will take a considerable amount of time to do. An alternative could therefore be to bundle this work in with the upcoming Giza release, and either not implement this part of the improvements in this sprint, or to implement a simpler version which does not try to give the detailed information presented in the dialog, merely some sort of dialog that explains that the publishing failed due to a specific reason.

# Todo

- Adam will write an issue describing the idea of introducing design tokens, and share with team, we decided not to start that work now.

# Non-action Conclusions

- We decided to start doing explicit up-front sprint planning for design sprints, described here: https://github.com/Joystream/atlas/issues/1309
- We are now a jour with design reviews on the product in production, and we have started doing more frequent design reviews on work that is in progress.
- Adam will start working on the design side of a new sprint to tackle the broken search experience.
- We decided that we will on an on-going basis tackle how to improve documentation of animations, which currently are a bit ad-hock.
- We decided that we will not go for the full-blown storage quota detection solution, since the query node support is not worth introducing now. We will instead introduce a more descriptive error dialogue when publishing transactions fail.
- We decided to start using a new logo for the studio side of Atlas.
- From now on, design review issues will describe a short list of required data queries that Klaudisz can review.
- Availability:
  - Diego is off next week, Mon 6th.
  - Klaudiusz off for a week, starting 13th of September.

![Screenshot 2021-09-01 at 14 54 21](https://user-images.githubusercontent.com/437292/131674655-2a5b9243-914e-4783-8894-6964693ff74a.png)


# Expected Releases

`None`

**Comment by bedeho** [comment]
# Meeting Date

Monday, September 7th, 2021.

_Note: meeting was postponed from yesterday due to Klaudiusz being unavailable._

# Participants

- Bedeho
- Klaudiusz
- Kuba
- Adam

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-912555346
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-912652404

# Agenda

- Review status of TODOs from last week.
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- Review new Gantt chart for inter-temporal coordination, and update based on new information.

# Todo

- Kuba will share suggestion for what to work on for remainder of next week, before which he is off for a week, most likely it will be some design debt on user input boxes.
- Kuba and Adam will add work to review different PX Grotesk alternatives for Atlas, and the tech team will deploy, top 3 alternatives for review.
- Klaudiusz will decide on who, if anyone, takes his place in the next Atlas meeting.
- Bedeho will review notes from prior discussions about introducing stitching.

# Non-action Conclusions

- From now on, the design weekly summary will include milestone estimates for pure design sprints.
- There is an open question about what Klaudiusz should tackle next, he wants to tackle the data layer issue by adding the stitching node, Bedeho was concerned about timing, Klaudiusz can do work on upload flow sprint for now, but wants to settle a timeline for this issue.
- We will not make a design token summary issue for now, rather tackle it in an ongoing way.
- Main remaining work is changes to my `My Videos` page, and player overlay, one click video launch, as art of upload flow sprint ...
Expected release time is Friday this week (10.)
- Discovery sprint finished 17th, as Diego is away.
- Rafal is away, coming back 13th.
- PX Grotesk issues: we should no longer use it in Atlas, possibly also in any other products because
 - Licensing issues for uses, e.g. in play. joystream.org,  or people forking.
 - Does not support Cyrillic, which is becoming an issue due to all the Russian language content.
 - Inter is an alternative.
 
![Screenshot 2021-09-07 at 14 38 37](https://user-images.githubusercontent.com/437292/132345852-0623893b-49c2-474d-8065-2402b78b49ec.png)


# Expected Releases

- Thursday 9th: Fixes to video content pages, and new build pipeline.

# Availability

- **Kuba:** is off 17th -> 27th September, back 28th September.
- **Klaudiusz:** is off 13th -> 17th September.

**Comment by bedeho** [comment]
# Meeting Date

Monday, September 13th, 2021.

# Participants

- Bedeho
- Kuba
- Adam

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-917417145
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-916931590

# Agenda

- Review status of TODOs from last week.
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- Review new Gantt chart for inter-temporal coordination, and update based on new information.

# Todo

- Kuba & Adam will review the NFT design brief ( https://github.com/Joystream/atlas/issues/1329)  and make a recommendation about how it should be tackled when we start work on it Mon 27th.
- Bedeho will finsih review of new search design today.

# Non-action Conclusions

- Kuba still expects Tuesday (14th) to be last day.
- Kuba & Adam decided to post-pone the work on revising the possible alternatives to PX Grotesk.
- Kuba has decided to work on the original goal, which was to tackle some user-input deficiencies.
- Kuba will start to use Loom, we will switch to PRO plan, so that we can go back and download all files ex-ante.
- We expect a new designer to join us on Wednesday, Nina (a product designer)

# Expected Releases

- `None`

# Availability

- **Kuba:** is off 17th -> 27th September, back 28th September.
- **Klaudiusz:** is off 13th -> 17th September.

**Comment by bedeho** [comment]
# Meeting Date

Monday, September 20th, 2021.

# Participants

- Bedeho
- Kuba
- Adam
- Klaudiusz

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-921989251
- **Tech:** `Not provided`

# Agenda

- Review status of TODOs from last week.
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- Review new Gantt chart for inter-temporal coordination, and update based on new information.

# Todo

`Fill in post-meeting`

# Non-action Conclusions

`Fill in post-meeting`

# Expected Releases

`Fill in post-meeting`

# Availability

- **Kuba:** is off 17th -> 27th September, back 28th September.
- **Nina** is off this full week.

**Comment by bedeho** [comment]
# Meeting Date

Monday, September 20th, 2021.

# Participants

- Bedeho
- Adam
- ~Kuba~
- ~Nina~
- Klaudiusz

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-921989251
- **Tech:** `Not provided`

# Agenda

- Review status of TODOs from last week.
- Review status of live milestones: https://github.com/Joystream/atlas/milestones
- Review new Gantt chart for inter-temporal coordination, and update based on new information.

# Todo

- Bedeho will share notes on viewer side membership in Atlas.
- Bedeho will share additional notes on restricting MVP NFT release.
- Adam review Bedeho notes and possibly schedule another meeting if required to clarify.
- Klaudiusz will look into restricting scope, reevaluating estimate and contributing on Discovery Sprint, which is quite delayed, and currently estimated to take 2-4 additional weeks.

# Non-action Conclusions

- Atlas needs time sufficient time to properly plan and prepare Giza release, this will cut into other expected Atlas release timelines.
- Discovery sprint is expected in 2-4 weeks, so quite delayed.
- Adam % team reviewed the NFT design brief and concluded
  - It was hard to identify a significantly reduced scope which still made sense.
  - Starting with introducing memberships on the consumer side of Atlas made sense.
  - It was probably effective to have many designers working in parallel.
  - MVP scope not yet identified.
  
# Gantt Chart

![Screenshot 2021-09-20 at 16 35 10](https://user-images.githubusercontent.com/437292/134020788-125fba31-3e1a-427d-a608-70e4734d2a71.png)

# Expected Releases

`None`

# Availability

- **Kuba:** is off 17th -> 27th September, back 28th September.
- **Nina** is off this full week.

**Comment by bedeho** [comment]
# Meeting Date

Monday, September 27th, 2021.

# Participants

- Bedeho
- Adam
- ~Kuba~
- Nina
- Klaudiusz

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-926836930
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-927115658

# Agenda

- Introduction to Nina, and review of how onboarding has gone.
- Giza release imposition on Atlas team
- Discuss status of NFT sprint planning, prior questions include
   - how to narrow scope productively
   - current thinking on viewer side account/membership.
- Review status of TODOs from last week.
- Review new Gantt chart
   - Update chart based on review of current progress
   - What is status of previously badly delayed discovery sprint: **Klaudiusz wants to restrict scope**
   - Introduce Giza

# Todo

- Bedeho will invite Nina to our Github.
- Bedeho will clean up the Github membership list for Atlas.
- Klaudiusz will make issue about different ways to automate updating Atlas featured content.
- Klaudiusz will make issue with suggestions for new sprints before NFT work can begin.
- Kuba does design review of upload flow sprint.

# Non-action Conclusions

- Adam & Nina will have additional meetings to review planning NFT, may involve shared meeting on Wed. Nina will continue learning about the topic in the interim.
- Klaudiusz suggest restricting scope of discovery sprint
  - remove search from category page
  - avoid having to update Orion to facilitate top watched videos this week and top channels this week, will take substantial work, payoff uncertain currently, alternatives are
    1. remove everything
    2. all time most watched, this still requires Orion intervention, but less
    3. make everything manual on application owner side.

    We agreed that c) is the best tradeoff. It will involve some amount of tech & community policy updates to allow them to make changes without waiting for Atlas dev team, but that is likely worth it going forward.
- We expected work to finish by Wed on Upload flow, and Kuba will do design review when back on Friday.
- We agreed to add category filter to search.
- It is not clear how to effectively occupy the Atlas dev team when a lot of the current sprints close.

# Gantt Chart

![Screenshot 2021-09-27 at 13 28 50](https://user-images.githubusercontent.com/437292/134899868-6959797a-7da7-4e50-b357-964472251d25.png)

# Expected Releases

- Upload flow improvements on Wed 29th.

# Availability

- Kuba comes back next week.

**Comment by bedeho** [comment]
# Meeting Date

Wednesday, October 6th, 2021.

# Participants

- Adam
- Kuba
- Nina
- Klaudiusz
- Bedeho

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-932330851
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-932735508

# Agenda

- Review Status of NFT design sprint scope.
    - Address unaddressed questions here: https://github.com/Joystream/atlas/issues/1329#issuecomment-929945502
    - Current thinking on viewer side account/membership
- Review status of TODOs from last week.
- Review new Gantt chart
  - Big question: when can tech side of NFT start at earliest?
  - Big action item here is: what to prioritise
- Pure tech topics with Klaudiusz
  - How to do community managed featuring infra: https://github.com/Joystream/atlas/issues/1425
  - Stale `Discover new channels` section on Home page

# Todo

- Bedeho answer questions in `Questions` tab of sheet.
- Kuba will be in charge to create issues on Atlas repo which cover all design tasks in NFT sprint which covers current scope and has estimates on hours needed for all issues, and then derive total estimate for completion based on how work is split among the three designers. Perhaps making a dependency diagram can help
  - https://user-images.githubusercontent.com/437292/133438763-0daf88e6-2101-42fd-9363-db9f5363610e.png
- Klaudiusz will make issue with suggestions for new sprints before NFT work can begin.
- Bedeho will review new issues around upload flow related design review.
- Klaudiusz will start start on introducing featuring in Orion.
- Klaudiusz will make issue describing staleness of `Discover new channels`, design team can review, and we can discuss solutions next meeting.

# Non-action Conclusions

- We will introduce usage of storage system for member assets in NFT release
 make note of complexity loading full storage integration in P2 as a result of this, may delay release.
- This should have implication for channel creation flow as well.
- We will possibly allow owner of video to have differentiated view of video, controlling NFT state, however, the design portion of this will be delayed until we are done with My Videos based control.
- Establishing connectivity with a full node it takes a lot of time, multiple seconds, this can be quit costly on the UX on viewer side, Klaudiusz will review options to resolve.
- Too early to say when engineering work can start on NFT sprint.
- A possible blocker for engineering work on NFTs is query node status, can be possible task for Atlas.
- Orion will be turned into the gateway node, and will have the featuring information.

# Gantt Chart

![Screenshot 2021-10-06 at 18 54 32](https://user-images.githubusercontent.com/437292/136249028-6126da0f-19df-43a6-8e3b-1bb0e7e5152e.png)

# Expected Releases

- Perhaps upload flow.

# Availability

**Full Team**

**Comment by bedeho** [comment]
# Meeting Date

Wednesday, October 11th, 2021.

# Participants

- Adam
- Kuba
- Nina
- Klaudiusz
- Bedeho

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-938671840
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-939303086

# Agenda

- Review Status of NFT design sprint scope.
- Drop support for video poster: https://github.com/Joystream/atlas/issues/1469
- Review status of TODOs from last week.
- Review new Gantt chart

# Todo

- Right after call: Bedeho answer questions in Questions tab of sheet.
- Kuba will share work model for NFT design sprint with Bedeho, likely by tomorrow.
- Bedeho will review the model.
- Klaudousz suggest three sprints that can be worked on by tech team before NFTs, wills hare.
- Adam will spend 0.5 day to redesign the player in light of changes agreed upon w.r.t. https://github.com/Joystream/atlas/issues/1469.
- Tech team will have a quick sprint to updated player based on Adam's work.

# Non-action Conclusions

- Kuba has proposed a model for the design progress on the.
- Klaudiusz fixed stale `Discover New Chanels`

# Gantt Chart

![Screenshot 2021-10-11 at 11 42 17](https://user-images.githubusercontent.com/437292/136769156-ece9bd0e-be7b-43e6-b7bd-aebbe250165a.png)
# Expected Releases

- Resolving https://github.com/Joystream/atlas/issues/1469.

# Availability

- Bartosz away 2 weeks starting Friday.

**Comment by bedeho** [comment]
# Meeting Date

Wednesday, October 18th, 2021.

# Participants

- Adam
- Kuba
- Nina
- Tomasz
- Klaudiusz
- Bedeho

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-944892412
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-944466710

# Agenda

- Process for community to manage featured content in Atlas: https://github.com/Joystream/atlas/issues/1486
- Briefly discuss curators and curator groups.
- Review status of TODOs from last week.
- Review new Gantt chart

# Todo

- Adam will provide guidelines for assisting the content curators to provide.
- Nina will provide first video on progression the membership screen.
- Kuba will provide estimates on major tasks for NFT MVP design sprint.
- Bedeho will incorporate into Gantt chart when Kuba has it available.
- Adam will provide plan for initial work in clearing design debt in design system.
- Designers will meet and decide how to onboard Tomek into current NFT design sprint.
- Adam will share link to survey about what alternative typeface.
- Klaudiusz will create cloud spreadsheet and invite everyone to publish their absence dates for visibility.

# Non-action Conclusions

None.

# Gantt Chart

![Screenshot 2021-10-18 at 12 02 34](https://user-images.githubusercontent.com/437292/137710525-0b934d49-9929-4b93-b21c-8a458fb8e43a.png)

# Expected Releases

None.

# Availability

- Kuba is off
- Tomek away Thursday & Friday
- Rafal is off Thursday, Friday & half of Wednesday
- Bartosz is off
- Internal Designer Conferce
    - Wed-Thursday, 15-17pm
    - Friday, 12pm-17pm

**Comment by bedeho** [comment]
# Meeting Date

Monday, October 25th, 2021.

# Participants

- Adam
- Kuba
- ~Nina~
- Tomasz
- Klaudiusz
- Bedeho

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-949687461
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-950312297

# Agenda

- Poor new content visibility: https://github.com/Joystream/atlas/issues/1507
- Review status of TODOs from last week.
- Review new Gantt chart.

# Todo

- Kuba will help with completion times for specific subtasks in sprint
  - Issuing
  - Member Account
  - Notifications
  - Player experiences
  - Purchasing Flow
- Bedeho will incorporate into Gantt chart when Kuba has it available.
- Tomasz & Kuba expect to have new iterations this week, Tomasz needs urgent review.

# Non-action Conclusions

- NFT
  - We will keep using link only for membership assets for the NFT release.
  - We will not introduce membership covers for the NFT release.
- We decided to go for `IBM Plex Sans`.
- Core product work on `Discovery` sprint is mostly done, the community side may have some delays to be ready.
- Unclear what Rafal & Diego should focus on next.
- We will introduce home hero section specific preview image as part of feature metadata.

# Gantt Chart

![Screenshot 2021-10-25 at 13 12 07](https://user-images.githubusercontent.com/437292/138685350-5b5de366-7aac-497a-814e-e7042d706b31.png)


# Expected Releases

- Possibly `Discovery` sprint.
- Possibly `Search` sprint.

# Availability

- Nina way today.
- Bartosz: not back until Nov. 2.

**Comment by bedeho** [comment]
# Meeting Date

Wednesday, November 3rd, 2021.

# Participants

- Adam
- Kuba
- Nina
- Tomasz
- Klaudiusz
- Bedeho

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-954887372
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-955480507

# Agenda

- Atlas developer onboarding: It would be nice if the Atlas product was more friendly to outsiders looking to contribute. This would primarily be achieved by enhancing the communication of the primary README.md and associated documents on the public repo. A combined effort across tech lead and design system leads could likely in a very short amount of time really raise our level on this, and it could open up near term participation from community developers, and eventually even designers. It would of course also help with onboarding of new Atlas related staff internal to Jsgenesis operations.
- Illustration style in Atlas is currently mixed. We use both the new and sleek style, for example for the failure screens, and the old bloated style, for example during membership creation, and also during demo videos. We should just make a whole sale switch over to the new style, and use it during exploration work as well, to avoid confusion.
- Can we have a standard intro photo for the beginning of all of the designer design iteration videos?
- How should design team progress beyond 05.11 deadline? Seems like best use of time is to revisit scope limitations and see what else can be tackles while NFTs are still fresh on everyones mind. This is also better if tech team will start tackling soon, rather than shifting focus.
- When will will tech team start tackling
  - Giza release
  - NFTs <== most appears to be done 05.11
- If part of tech team is still blocked, here are two tasks I can suggest
  - internationalisation: https://github.com/Joystream/atlas/issues/1531
  - speed control in player: https://github.com/Joystream/atlas/issues/1503
  - in-app feedback: https://github.com/Joystream/atlas/issues/879  
- Review status of TODOs from last week.
- Review new Gantt chart.

# Todo

`Fill post-meeting`

# Non-action Conclusions

`Fill post-meeting`

# Gantt Chart

`Fill in later`

# Expected Releases

`Fill post-meeting`

# Availability

`Fill post-meeting`

**Comment by bedeho** [comment]
# Meeting Date

Wednesday, November 3rd, 2021.

# Participants

- Adam
- Kuba
- Nina
- Tomasz
- Klaudiusz
- Bedeho

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-954887372
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-955480507

# Agenda

- Illustration style in Atlas is currently mixed. We use both the new and sleek style, for example for the failure screens, and the old bloated style, for example during membership creation, and also during demo videos. We should just make a whole sale switch over to the new style, and use it during exploration work as well, to avoid confusion.
- How should design team progress beyond 05.11 deadline? Seems like best use of time is to revisit scope limitations and see what else can be tackles while NFTs are still fresh on everyones mind. This is also better if tech team will start tackling soon, rather than shifting focus.
- When will will tech team start tackling
  - Giza release
  - NFTs <== most appears to be done 05.11
- If part of tech team is still blocked, here are two tasks I can suggest
  - internationalisation: https://github.com/Joystream/atlas/issues/1531
  - speed control in player: https://github.com/Joystream/atlas/issues/1503
  - in-app feedback: https://github.com/Joystream/atlas/issues/879  
- Review status of TODOs from last week.
- Review new Gantt chart.

# Todo

- Adam and Klaudiusz will work together for 1-2 days improving the onboarding for, primarily, developers on the Atlas repository
- Adam will review Discovery delivery.
- Adam will review Search delivery.
- Kuba is going to test the new iteration of the issuance flow on users inside Netguru in order to select what variation to go for.

# Non-action Conclusions

- Main documentation for design will be incorporated into Atlas repo and removed from standalone atlas design repo.
- Tomasz will be working on notifications next.
- Nina will be working on player page next.
- Designers are fully occupied, and will remain so for a time going forward.
- After the designers have all finished their independent sprints, largely based on building prototypes, there will be a substantial delay until the developers can start to build, as the preparation and handoff - lead by Adam, will take almost an equivalent amount of time as the initial design stages.
- Klaudiusz has developed some good initial understanding of the `Giza` release, however is not done yet, and estimates that by mid next week the tech team may be able to start working on this.
- Klaudiusz having a hard time keeping up with design work to provide ongoing feasibility constraints on the tech side, but we did not find a cost-effective way of dealing with this.

# Gantt Chart

![Screenshot 2021-11-04 at 11 13 44](https://user-images.githubusercontent.com/437292/140296277-9223c5a5-4d00-4b46-bc0c-42df1d11339e.png)

# Expected Releases

- Search sprint will be released this.

# Availability

- 11 - 12.11 most team is off

**Comment by bedeho** [comment]
# Meeting Date

Wednesday, November 8rd, 2021.

# Participants

- Adam
- Kuba
- Nina
- Tomasz
- Klaudiusz
- Dmitry
- Bedeho

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-962020856
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-962443085

# Agenda

- Introduce Dmitry to team.
- Giza planning progress.
- Review status of TODOs from last week.
- Review new Gantt chart.

# Todo

- Nina expects to have new update ready today on NFT states.
- Nina expects to have new update ready on player page by Wednesday.
- Kuba expects to have a new update by Friday.
- Tomasz expects to have a new update by Wednesday
- Adam & Klaudiusz post-pone their work on Atlas repo onboarding to next week, due to this week being shorter.
- Klaudiusz start detailed planning of Giza execution today, expects work to start on Wednesday
ing the onboarding for, primarily, developers on the Atlas repository

# Non-action Conclusions

- Klauiudsz has assigned the tech team to focus on the following while the NFT work is landing
  - moving schema stitching out of Atlas and into Orion
  - improving Figma icon export
  - reorganising component folder
- We Open auctions serve the purpose, previously partially discussed on RocketChat, of allowing users to passively accept bids in public that are committing. Design work remains on what initial state we should nudge the user into, and also whether we should introduce functionality to allow recipient of a transaction to specify default initial non-idle state.

# Gantt Chart

![Screenshot 2021-11-08 at 13 45 26](https://user-images.githubusercontent.com/437292/140744403-83bdcb25-6d42-4031-8df9-2348a6df61c4.png)

# Expected Releases

- Discovery blocked by community content selection.
- Search is ready.

# Availability

- Thursday and Friday most of the Polish side of the team will be off.

**Comment by bedeho** [comment]
# Meeting Date

Wednesday, November 15th, 2021.

# Participants

- Adam
- Kuba
- Nina
- Tomasz
- Klaudiusz
- Dmitry
- Bedeho

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-968079562
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-968708361

# Agenda

- Figma publicly.
- Discovery release status?
- Review status of TODOs from last week.
- Review new Gantt chart.
- ETA on NFT dev work is still: Friday 3rd December?

# Todo

- Adam will prepare conclusion about when we can start sharing completed designs to the public in dedicated public read-only boards.
- Kuba will have update & questions for Bedeho today
- Tomasz will have an update for Bedeho today.
- Adam & Klaudiusz hvae their meeting late this week about improving repo onboarding.
- Klaudiusz will focus on completing Giza planning.
- There will be Atlas team retro meeting on Wednesday run by Dmitry.
- Dmitry will present alternatives for analytics integration into Atlas.

# Non-action Conclusions

- Just sharing working Figma boards, as is, would be too disruptive for designers and also cause slowness, we will instead try to have dedicated viewing boards instead.
- Discovery release is being held up by community.
- Design team will review whether design works well with a wide range of number of decimals needed in balance quantity display.
- Full tech team is tackling a lof smaller backlog issues.

# Gantt Chart

![Screenshot 2021-11-15 at 13 56 09](https://user-images.githubusercontent.com/437292/141785777-0f44e659-d13a-4bed-ac56-278cb579a783.png)

# Expected Releases

`nothing`

# Availability

`nothing`

**Comment by bedeho** [comment]
# Meeting Date

Wednesday, November 22nd, 2021.

# Participants

- Adam
- Kuba
- Nina
- Tomasz
- Klaudiusz
- Dmitry
- Bedeho

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-974120858
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-974142826

# Agenda

- JOY asset logo
- Review status of TODOs from last week.
- Review new Gantt chart.
  - Design progress last week and plans for this week:
    - user tests results/ or planned experiments
    - handover dependencies
    - work in progress comments and timelines
  - Dev progress last week and plans for this week:
    - progress towards release plans
    - release ETAs
    - dependencies on other dev squads/ services
- AOB - any other business that impacts the work of Atlas team

# Todo

- Adam will prepare conclusion about when we can start sharing completed designs to the public in dedicated public read-only boards.
- Klauiusz will potentially come up with structure for README.
- Tomasz expects Notifications udpate by Friday.
- Dmitry will book a meeting to discuss how to do user testing with MAze and possibly Lookback.

# Non-action Conclusions

- JOY asset
  - will be tackled by Atla design team
  - potentially including community to make final decision, and Dmitry will help with Maze.
  - will bet takcled right away
- Repo onboarding conclusion:
  - Klauiusz will come up with structure for README
- User testing, two kinds will be done
  - quality tests: 20 people, no advice and input, we measure key action metrics. In the end we ask them what they thhink aboout ut, but this does not give us
  - quantity tests: fully implement everything in the product
  Dmitry thinks doing a pure functional test using Maze, with one flow per varition. you ask them in the end, which one do you preper. We can also use Lookback. Dmitry thinks 5-7
- Dev work will already start this week on NFT epic.

# Gantt Chart

![Screenshot 2021-11-22 at 11 55 02](https://user-images.githubusercontent.com/437292/142849248-ee31bc21-ed61-4703-8a32-b0708b758909.png)

# Expected Releases

`nothing`

# Availability

`nothing`

**Comment by bedeho** [comment]
# Meeting Date

Wednesday, November 29nd, 2021.

# Participants

- Adam
- Kuba
- Nina
- Tomasz
- Klaudiusz
- Dmitry
- Bedeho

# Weekly Summaries

- **Design:** https://github.com/Joystream/atlas/issues/816#issuecomment-980561631
- **Tech:** https://github.com/Joystream/atlas/issues/815#issuecomment-981087282

# Agenda

- Review status of TODOs from last week.
- How are we triaging new issues, like this => https://github.com/Joystream/atlas/issues/1659
- How can we setup detection of specific problems in Atlas UX: e.g. like infra problems.
- Review new Gantt chart.

# Todo

`Fill in later`

# Non-action Conclusions

`Fill in later`

# Gantt Chart

`Fill in later`

# Expected Releases

`nothing`

# Availability

`nothing`

**Comment by bedeho** [comment]
This meeting log is no longer needed.
